sechub-pds-examples/gosec/docker/loadbalancer/nginx.conf

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: MIT
+
+# Based on config from: https://stackoverflow.com/a/54388725
+
+worker_processes  1;
+
+events {
+  worker_connections  1024;
+}
+
+http {
+  access_log /dev/stdout;
+  error_log /dev/stderr;
+
+  server {
+    listen 8444 ssl;
+    server_name localhost;
+
+    ssl_certificate /certificates/localhost.cert;
+    ssl_certificate_key /certificates/localhost.key;
+
+    ssl_protocols TLSv1.2 TLSv1.3;
+    ssl_prefer_server_ciphers on;
+    ssl_dhparam /certificates/certsdhparam.pem;
+    ssl_ecdh_curve secp384r1;
+
```

Comment:
I think the Diffie-Hellman configuration is good.
If it is not too restrictive, you could consider the following ciphers:
```
ssl_ciphers EECDH+AESGCM:EDH+AESGCM;
```

Reply:
I tested the `ssl_ciphers` and it works. As a result, I included your suggestion.